editors/vscode/pnpm-lock.yaml

```diff
@@ -1,7 +1,7 @@
 lockfileVersion: '6.0'
 
 settings:
-  autoInstallPeers: true
+  autoInstallPeers: false
```

Comment:
How did you edit this 🤔 ?

Reply:
it is update after I run `pnpm i`, I reset them already.